components/layout/layout.component.ts

```diff
@@ -41,12 +41,6 @@ export class HsLayoutComponent {
       element: elementRef.nativeElement,
       innerElement: '.hs-map-container',
     });
-    if (HsConfig.sidebarPosition === 'left') {
-      this.HsLayoutService.contentWrapper.classList.add('flex-reverse');
-      this.HsLayoutService.sidebarRight = false;
-    } else if (HsConfig.sidebarPosition != 'invisible') {
-      HsConfig.sidebarPosition = 'right';
-    }
 
     if (this.HsConfig.importCss == undefined || this.HsConfig.importCss) {
```

Comment:
What would be the use case when we dont want to import css?

Reply:
if hslayers breaks some styles in the container application